app/views/trainees/itt_start_dates/edit.html.erb

```diff
@@ -0,0 +1,25 @@
+<% text = I18n.t("components.page_titles.trainees.start_date.edit") %>
+
+<%= render PageTitle::View.new(text: text, has_errors: @start_form.errors.present?) %>
+
+<%= content_for(:breadcrumbs) do %>
+  <%= render GovukComponent::BackLink.new(
+    text: "Back",
+    href: trainee_path(@trainee),
+  ) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <%= register_form_with(model: @start_form, url: trainee_course_details_itt_start_date_path(@trainee), local: true) do |f| %>
+      <%= f.govuk_error_summary %>
+        <%= f.govuk_radio_buttons_fieldset(:date_string, legend: { text: text, tag: "h1", size: "l" }, classes: "start-date", hint: {text: "The start date of the Initial Teacher Training portion of their course. "}) do %>
```

Comment:
Can we have any lines going beyong 120 characters move into a new line?

Reply:
rewrite that bit as there isn't a radio button and move text to locale file

Reply:
> 120 characters 

That should really be in a linter craziness otherwise our lack of coding standards will drive insane to sane and vice versus.

Carded as 

https://trello.com/c/6U4OLa3i/2330-120-characters-limit-linter